docs/src/lib/indexing.md

```diff
@@ -133,9 +135,9 @@ so it is unsafe to use it afterwards (the column length correctness will be pres
                        `v` must be an `AbstractMatrix` or an `AbstractDataFrame`
                        (in the latter case column names must match);
 
-Note that only `df[!, col] = v` and `df.col = v` can be used to add a new column to a `DataFrame`.
-In particular as `df[:, col] = v` is an in-place operation it does not add a column `v` to a `DataFrame` if `col` is missing
-(an error is thrown if such operation is attempted).
+Note that `df[!, col] = v`, `df.col = v` and `df[:, col] = v` can be used to add a new column to a `DataFrame`.
```

Comment:
Maybe this paragraph isn't needed anymore? This just repeats the general behavior applied to a special case. If you want to stress what is special about non-existing columns, better mention the fact that conversion may happen when the column exists, but (obviously) not when it doesn't.

Reply:
I removed it. It was just a duplication indeed.